modules/microshift-install-rhel-image-mode-csi-vgloop.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-csi_{context}"]
+= Configuring container storage for the bootc image and the {microshift-short} CSI
+
+Create and allocate an additional LVM volume group loop device to be shared with the container and used by the {microshift-short} Logical Volume Manager (LVM) Container Storage Interface (CSI).
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have SUDO permissions configured.
+
+.Procedure
+
+. Create a file to be used for LVM partitioning by running the following command:
++
+[source,terminal]
+----
+VGFILE=/var/lib/microshift-lvm-storage.img
+
+sudo losetup -f "${VGFILE}"
+----
+
+. Configure the partition as a loop device by running the following command:
++
+[source,terminal]
+----
+VGSIZE=1G
+
+sudo truncate --size="${VGSIZE}" "${VGFILE}"
+----
+
+. Query the loop device name by running the following command:
++
+[source,terminal]
+----
+$ VGLOOP=$(losetup -j ${VGFILE} | cut -d: -f1)
+----
+
+. Create a free volume group on the loop device by running the following command:
++
+[source,terminal]
+----
+$ sudo vgcreate -f -y rhel "${VGLOOP}"
+----
+//according to the MicroShift CSI driver requirements described in Storage Configuration
+//TODO are we missing steps?
+
+. Share the device with the newly created containers by using the following steps:
++
+.. Detach the loop device by running the following command:
++
+[source,terminal]
+----
+$ sudo losetup -d "${VGLOOP}"
+----
+
+.. Delete the LVM volume group file by running the following command:
++
+[source,terminal]
+----
+$ sudo rm -f "${VGFILE}"
+----
```

Comment:
These steps are for deleting the device and volume file (cleanup). We should put it in a separate section or note.

Reply:
Are there any other cleanup steps we should add for networking or anything else?

Reply:
No, I don't think there's anything else.